gpuselect/__init__.py

```diff
@@ -8,27 +10,29 @@
 import ConfigParser
 logger = logging.getLogger("gpuselect")
 
+
 def get_gpu(gpu_weight, mem_weight):
     n_gpus = gs.n_gpus()
     G, M = [], []
-    for i in xrange(n_gpus):
+    for i in range(n_gpus):
```

Comment:
No need to change it but FYI if you wanna be nicely compatible with both Pythons there's: `six.moves.range` (which is the same as `six.moves.xrange`).